src/pages/Explorer/Category/Category.svelte

```diff
@@ -29,13 +72,21 @@
   </div>
 
   {#each items as item (item)}
-    <div class="item btn">
+    {#if isMain && item.first}
+      <div class="postdate c-waterloo">{item.postdate}</div>
+    {/if}
```

Comment:
```suggestion
    <slot name="before" />
```

Reply:
@DmitriVanGuard I don't understand this comment, Can you describe more?